privacy-protections/click-to-load/index.html

```diff
@@ -6,33 +6,35 @@
   <title>Facebook click to load</title>
 </head>
 <body>
-  <!-- Load the facebook SDK in the recommended way -->
-  <script async defer crossorigin="anonymous" src="https://connect.facebook.net/en_US/sdk.js?XFBML=false"></script>
+<!-- Load the facebook SDK in the recommended way -->
+<div id="fb-root"></div>
+<script async defer crossorigin="anonymous" src="https://connect.facebook.net/en_US/sdk.js#xfbml=1&version=v14.0" nonce="1SgoaWbh"></script>
 <p><a href="../../">[Home]</a> ↣ <a href="../">[Privacy Protections Tests]</a> ↣ <strong>[Facebook click to load]</strong></p>
 
-<h2> About ClickToLoad Tests</h2>  
-<p>This page embeds the facebook SDK, and several facebook iFrames. It also attempts to create 
-a variety of social elements using the SDK.</P>
+<h2> About ClickToLoad Tests</h2>
+<p>This page embeds the facebook SDK, and several facebook iFrames. It also attempts to create
```

Comment:
Nit: It kind of bugs me how Facebook is capitalised inconsistently, would you mind making it consistent? (Elsewhere too.)

Reply:
👍